apiserver/facades/client/application/application.go

```diff
@@ -678,9 +679,11 @@ func checkMachinePlacement(backend Backend, args params.ApplicationDeploy) error
 	return nil
 }
 
-// ApplicationSetSettingsStrings updates the settings for the given application,
+// applicationSetSettingsStrings updates the settings for the given application,
```

Comment:
This didn't need exporting. Drive-by to make it private for congruence with the other methods.